module-router.c

```diff
@@ -2034,11 +2034,11 @@ int pa__init(pa_module *m) {
             PA_HOOK_LATE + 30, (pa_hook_cb_t) hook_callback_source_output_put, u);
     u->h->hook_slot_source_output_unlink = pa_hook_connect(&m->core->hooks[PA_CORE_HOOK_SOURCE_OUTPUT_UNLINK],
             PA_HOOK_LATE + 30, (pa_hook_cb_t) hook_callback_source_output_unlink, u);
-    u->h->hook_slot_sink_new = pa_hook_connect(&m->core->hooks[PA_CORE_HOOK_SINK_NEW], PA_HOOK_LATE + 30,
+    u->h->hook_slot_sink_new = pa_hook_connect(&m->core->hooks[PA_CORE_HOOK_SINK_FIXATE], PA_HOOK_LATE + 30,
```

Comment:
Hi Matt-san,

Thank you very much for providing this patch.

Recently I was also reported that bluetooth audio did not become audible in AGL demo which integrated this module with AudioManager.
According to the log that I received module-router sent empty sink name and AudioManager crashed with assertion.

I guess we are in same situation and hopefully this patch improve it.
I will apply this patch and check.

Thanks.

Best regards,
Toshiaki